src/scenes/Reports/ReportsList.tsx

```diff
@@ -1,7 +1,160 @@
+      if (!activeLocale) {
+        return [];
+      }
+
+      return [
+        {
+          key: 'report',
+          name: strings.REPORT,
+          type: 'string',
+        },
+        {
+          key: 'status',
+          name: strings.STATUS,
+          type: 'string',
+        },
+        {
+          key: 'year',
+          name: strings.YEAR,
+          type: 'string',
+        },
+        {
+          key: 'modifiedBy',
+          name: strings.LAST_EDITED_BY,
+          type: 'string',
+        },
+        {
+          key: 'submittedBy',
+          name: strings.SUBMITTED_BY,
+          type: 'string',
+        },
+        {
+          key: 'submittedTime',
+          name: strings.DATE_SUBMITTED,
+          type: 'string',
+        },
+      ];
+    },
+    [activeLocale, strings]
+  );
+
+  const dispatchSearchRequest = useCallback(
+    (locale: string | null, search: SearchNodePayload, sortOrder: SearchSortOrder) => {
+      if (!currentParticipantProject?.id) {
+        return;
+      }
+
+      const request = dispatch(
+        requestListAcceleratorReports({
+          includeFuture: true,
+          locale: locale ?? undefined,
+          projectId: currentParticipantProject.id.toString(),
+          search,
+          sortOrder,
+        })
+      );
+      setListAcceleratorReportsRequestId(request.requestId);
+    },
+    [currentParticipantProject?.id, dispatch]
+  );
+
+  const fuzzySearchColumns = useMemo(() => ['report'], []);
+
+  const availableYears = useMemo(() => {
+    const years = acceleratorReports.map((report) => report.startDate?.split('-')?.[0]);
+    const uniqueYears = Array.from(new Set(years));
+    return uniqueYears;
+  }, [acceleratorReports]);
+
+  const featuredFilters: FilterConfigWithValues[] = useMemo(() => {
+    const rejectedStatus = activeLocale ? (isAcceleratorRoute ? strings.UPDATE_REQUESTED : strings.UPDATE_NEEDED) : '';
+    const filters: FilterConfigWithValues[] = [
+      {
+        field: 'status',
+        label: strings.STATUS,
+        options: AcceleratorReportStatuses,
+        pillValueRenderer: (values: (string | number | null)[]) =>
+          values.map((value) => (value === 'Rejected' ? rejectedStatus : value)).join(', '),
+        renderOption: (value: string | number) => (value.toString() === 'Rejected' ? rejectedStatus : value.toString()),
+      },
+      {
+        field: 'year',
+        label: strings.YEAR,
+        options: availableYears,
+      },
+    ];
+
+    return activeLocale ? filters : [];
+  }, [activeLocale, availableYears, isAcceleratorRoute]);
+
+  return (
+    <Card style={{ display: 'flex', flexDirection: 'column', paddingLeft: 0, paddingRight: 0 }}>
+      <Grid container sx={{}}>
+        <Grid item xs={12} paddingLeft={3}>
+          <Typography color={theme.palette.TwClrTxt} fontSize='20px' fontWeight={600} lineHeight='28px'>
+            {strings.REPORTS}
+          </Typography>
+        </Grid>
```

Comment:
Might be worth checking with design, but wondering if we could leave this out, since the page has a title of Reports, the tab is called Reports, and there's this card header of Reports

Reply:
I agree this feels overly redundant on this tab. I think Design opted for consistency over limiting redundancy in this case, but I can check.